zedd-app/src/components/SettingsDialog.tsx

```diff
@@ -218,17 +220,53 @@ export const SettingsDialog = observer(
                 }}
               />
             </Grid>
-
+            <Grid item xs={4}>
+              <FormLabel>Clarity Vacation Account</FormLabel>
+            </Grid>
+            <Grid item xs={8}>
+              <ClarityTaskSelect
+                clarityState={clarityState}
+                value={settings.vacationClarityTaskIntId}
+                onChange={(newIntId) => (settings.vacationClarityTaskIntId = newIntId)}
+              />
+            </Grid>
             <Grid item xs={4}>
               <FormLabel>Clarity Holiday Account</FormLabel>
             </Grid>
             <Grid item xs={8}>
               <ClarityTaskSelect
                 clarityState={clarityState}
-                value={settings.urlaubClarityTaskIntId}
-                onChange={(newIntId) => (settings.urlaubClarityTaskIntId = newIntId)}
+                value={settings.holidayClarityTaskIntId}
+                onChange={(newIntId) => (settings.holidayClarityTaskIntId = newIntId)}
               />
             </Grid>
+            <Grid item xs={4}>
+              <FormLabel>Location</FormLabel>
+            </Grid>
+            <Grid item xs={8}>
```

Comment:
We need to take into account that public holidays are different in different german states. This information is available via the API in the returned "counties" field. I would suggest saving "DE-BW" or similar in the settings.